doc/sphinx/addendum/extraction.rst

```diff
@@ -1,786 +0,0 @@
-.. _extraction:
```

Comment:
Are you simply deleting a bunch of the doc files?  That seems like a very bad idea.  I don't see why it's necessary in order to split stdlib into its own repo.